### PR TITLE
Create section 13 of controller Admin guide

### PR DESCRIPTION
**Pull request by ianf77**
Added assembly and module

Create section 13 of the Controller admin guide

https://issues.redhat.com/browse/AAP-17268

### CODE REVIEW

File: downstream/modules/platform/proc-controller-set-up-prometheus.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.Procedure
+. In the Prometheus configuration file (typically `prometheus.yml`), specify a `<token_value>`, a valid username and password for an {ControllerName} user that you have created, and a `<controller_host>`.
++
+[NOTE]
+====
+Alternatively, you can provide an OAuth2 token (which can be generated at `/api/v2/users/N/personal_tokens/`). 
+By default, the configuration assumes a user with username=`admin` and password=`password`.
+====
++
+Using an OAuth2 Token, created at the `/api/v2/tokens` endpoint to authenticate prometheus with {ControllerName}, the following example provides a valid scrape configuration if the URL for your {ControllerName}'s' metrics endpoint is `https://controller_host:443/metrics`.
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-controller-set-up-prometheus.adoc:20]: Should prometheus be capitalized? "authenticate Prometheus with..."

Comment (michellemacrh) [inline downstream/modules/platform/proc-controller-set-up-prometheus.adoc:20]: Is this a typo: the URL for your {ControllerName}'s' - should the second quote be removed?

Comment (ianf77) [reply downstream/modules/platform/proc-controller-set-up-prometheus.adoc:20]: I think it's a product name. it's captialised in the Prometheus documentation

**Review by michellemacrh — COMMENTED** [review]
A few small suggestions

**Comment by michellemacrh** [comment]
LGTM!

**Pull request merged**